welcome/learn_more_about_openshift.adoc

```diff
@@ -8,33 +8,42 @@ toc::[]
 
 Use the following sections to find content to help you learn about and better understand {product-title} functions:
 
+[id="support"]
+== Learning and support
+
+[options="header",cols="2*"]
+|===
+| Learn about {product-title} |Optional additional resources
+
+|link:https://www.openshift.com/learn/whats-new[What's new in {product-title}]
+|link:https://www.openshift.com/blog?hsLang=en-us[OpenShift blog]
+
+|link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[{product-title} life cycle]
+
+|link:https://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[OpenShift Interactive Learning Portal]
+|link:https://access.redhat.com/articles/4217411[OpenShift Knowledgebase articles]
+
+| xref:../support/getting-support.adoc#getting-support[Getting Support]
+| xref:../support/gathering-cluster-data.adoc#gathering-data[Gathering data about your cluster]
+
 [id="architecture"]
 == Architecture
 
-[options="header",cols="3*"]
+[options="header",cols="2*"]
 |===
-| Learn about {product-title} |Plan an {product-title} deployment |Optional additional resources
+| Learn about {product-title} |Optional additional resources
 
 | link:https://www.openshift.com/blog/enterprise-kubernetes-with-openshift-part-one?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Enterprise Kubernetes with OpenShift]
 | link:https://access.redhat.com/articles/4128421[Tested platforms]
-| link:https://www.openshift.com/blog?hsLang=en-us[OpenShift blog]
 
 | xref:../architecture/architecture.adoc#architecture[Architecture]
 | xref:../security/container_security/security-understanding.adoc#understanding-security[Security and compliance]
-| link:https://www.openshift.com/learn/whats-new[What's new in {product-title}]
 
-|
 | xref:../networking/understanding-networking.adoc#understanding-networking[Networking]
-| link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[{product-title} life cycle]
+| xref:../
```

Comment:
To be added

Reply:
Maybe [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/networking/ovn-kubernetes-network-plugin#ovn-kubernetes-architecture-con)